File: Application/WebAPI/Data/Models/OrderPackage.cs
using System;

namespace WebAPI.Data.Models
{
    public class OrderPackage : Entity
    {
        public int User_Id { get; set; } 
        public int Status_Id { get; set; }
        public int Timeline_Id { get; set; }
        public int Price_d { get; set; }
        public Guid PublicId { get; set; }
        public string Description { get; set; }
        public string Notes { get; set; }
        public bool IsFavourite { get; set; }
        public bool IsCompleted { get; set; }
    }
}

File: Application/WebApp/src/app/models/order-package.ts
import { Entity } from './Entity';
import { Order } from './order';

export interface OrderPackage extends Entity {
  user: number;
  status: string;
  timeline: [string, [Date]];
  price: [string, [number]];
  orders: [string, [Order]];
  publicId: string;
  description: string;
  notes: string;
  isFavourite: string;
  isCompleted: string;
}
